### DISCUSSION TITLE
Docker + Scheduled Tasks

### DISCUSSION THREAD

**Discussion by stavros-k** [outline/outline#5622] [Self Hosting] [answered]
Are the scheduled tasks "required" to be configured?

Will those tasks get eventually executed without `cron`? 
Or will those tasks get forever "stuck" in the queue.

Based on the docs I can safely assume that I have to setup the cron, and its not running inside the container. Is that right?

Thanks!

**Comment by tommoor** [comment, marked as answer]
> Are the scheduled tasks "required" to be configured?

The app will work without but old data will pile up and certain functionality will never be triggered.

> Based on the docs I can safely assume that I have to setup the cron, and its not running inside the container. Is that right?

That's correct.

**Reply by fantinialex** [reply]
Hi!

I came across this conversation and it made me curious: is there a reason why this isn’t handled directly inside the Docker container?
Just wondering why the scheduled tasks aren’t managed within the image itself.

Thanks!

**Reply by tommoor** [reply]
To allow horizontal scaling of the container. You don't want every server initiating the scheduled job

**Reply by fantinialex** [reply]
Ahh, that makes total sense. I didn’t think about it because I’m only running a single instance 😅
Thanks a lot for the clarification and the super quick reply!
